src/components/section/MobileHeader.jsx

```
@@ -0,0 +1,151 @@
+
+  const toggleMenu = () => {
+    setIsMenuOpen(!isMenuOpen);
+  };
+
+  return (
+    <div className='mobile-header-container'>
+      <div className="mobile-header-content">
+        <a
+          className="hamburger-button"
+          onClick={toggleMenu}
+        >
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)